peps/pep-0770.rst

```diff
@@ -139,6 +139,29 @@ These decisions mean this PEP is capable of supporting any SBOM standard
 and does not favor one over the other, instead deferring the decision to
 producing projects and tools and consuming user tooling.
 
+Adding data to Python packages without new metadata versions
+------------------------------------------------------------
+
+The rollout of a new metadata version and field requires that many different
+projects and teams need to adopt the metadata version in sequence to avoid
+widespread breakage. This effect usually means a substantial delay in how
+quickly users and tools can start using new packaging features.
+
+For example, a single metadata version bump requires
+updates to PyPI, various ``pyproject.toml`` parsing and schema projects,
+the ``packaging`` library, wait for releases, then ``pip`` and other installers
+need to bundle the changes to ``packaging`` and release, then build backends can
+begin emitting the new metadata version, again wait for releases, and only then
+can projects begin using the new features. Even with this careful approach it's
+not guaranteed that tools won't break on new metadata versions and fields.
+
+To avoid this delay, this PEP proposes a new mechanism ``[additional-files]``
```

Comment:
I think it's worth saying it isn't _just_ about getting this out faster as that feels like this is being done to cut a corner instead of doing what we think is the right approach.

```suggestion
To avoid this delay, simplify overall how to include SBOMs, and give flexibility to build back-ends, this PEP proposes a new mechanism ``[additional-files]``
```

Reply:
Great idea, thanks all! I've adopted this phrasing in https://github.com/python/peps/pull/4283/commits/502496bb37912d55b844c7c00cdb2a5f70a7867f and included the reasoning for not using `[project]`.